source/unified-test-format/unified-test-format.rst

```diff
@@ -1903,9 +1933,9 @@ It supports the following arguments:
   the loop MUST terminate and raise the error/failure (i.e. the
   error/failure will interrupt the test).
 
-- ``storeErrorsAsEntity``: if specified, the runner MUST capture errors
-  arising during sub-operation execution and append a document with error
-  information to the array stored in the specified entity.
+- ``storeErrorsAsEntity``: Optional string. If specified, the runner MUST
```

Comment:
For consistency with other argument lists, I went through and added: `[Optional] <type>`